src/include/network/lan/address.rb

```diff
@@ -1593,14 +1546,72 @@ def AddressDialog
 
   private
 
-    def get_selected_bridges
+    # Checks if the current LanItem is a bridge or an enslaved interface part
+    # of a bridge. In case that some configuration is not correct then adapt it
+    # if the user accepts.
+    def check_bridge_ports_config
+      return false unless LanItems.type == "br"
+
+      ports = LanItems.bridge_ports.split(" ").select { |p| old_bridge_port_config?(p) }
+
+      unless ports.empty?
+        ports.each { |p| configure_as_bridge_port(p) } if fix_bridge_port_config?(ports)
```

Comment:
you really want to raise a popup for every old fashioned bridge port? Poor SLE-11 user with 10 ports bridge ;-)